src/app/CASEClient.h

```diff
@@ -34,7 +34,6 @@ struct CASEClientInitParams
     Messaging::ExchangeManager * exchangeMgr                           = nullptr;
     FabricTable * fabricTable                                          = nullptr;
     Credentials::GroupDataProvider * groupDataProvider                 = nullptr;
-    Optional<ReliableMessageProtocolConfig> mrpLocalConfig             = Optional<ReliableMessageProtocolConfig>::Missing();
```

Comment:
@tcarmelveilleux if you feel this is not compatible enough, we could leave the field and either ignore it or start failing if it's set to to a non-missing value....

Reply:
If missing, it should use the local config.

Reply:
Or I guess if it's set to something other than `GetLocalMRPConfig()` at the point when init happens.

Reply:
> If missing, it should use the local config.

That is already the behavior, yes.